contracts/pegged/tokens/MintSwapCanonicalToken.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity >=0.8.9;
+
+import "./MultiBridgeToken.sol";
+
+/**
+ * @title Canonical token that supports multi-bridge minter and multi-token swap
+ */
+contract MintSwapCanonicalToken is MultiBridgeToken {
+    // each bridge token.balanceOf(this) tracks how much that bridge has already swapped
+    mapping(address => uint256) public tokenSwapCap; // each bridge token -> swap cap
```

Comment:
nit: term swapCap might cause confusion with swapBridgeForCanonical function, ie. someone might assume the cap is per swap call amount

Reply:
changed to totalSwapCap